examples/step-69/step-69.cc

```diff
@@ -2863,9 +2867,11 @@ namespace Step69
     // At this point we can return from the <code>output()</code> function
     // and resume with the time stepping in the main loop - the thread will
     // run in the background.
-    if (!asynchronous_writeback)
+    if (asynchronous_writeback)
       {
+#ifdef DEAL_II_WITH_THREADS
         background_thread_state = std::async(std::launch::async, output_worker);
+#endif
```

Comment:
would you mind adding something like
```
...
#else
  AssertThrow(false, ExcMessage("\"asynchronous_writeback\" was set to true but deal.II was built without thread support (\"DEAL_II_WITH_TREADS=false\")."));
#endif
```

Reply:
(and improve the error message a bit :smile: )

Reply:
I added the assert, I didn't change your message, I think that it is good ;)

Reply:
The line of the assert can not be break to 80 columns, I think that is because it is a macro